lib/sync-records.ts

```diff
@@ -25,24 +27,20 @@ export const syncWorkbookRecords = async ({
   organizationId: string;
   spaceType: SpaceType;
 }) => {
-  const accessToken = await getAccessToken();
-
   const employeeRecords = await getRecordsByName({
     userId,
-    accessToken,
     workbookName: process.env.ONBOARDING_WORKBOOK_NAME as string,
     sheetName: "Employees",
     spaceType,
   });
   const jobRecords = await getRecordsByName({
     userId,
-    accessToken,
     workbookName: process.env.ONBOARDING_WORKBOOK_NAME as string,
     sheetName: "Jobs",
     spaceType,
   });
 
-  const totalRecords = employeeRecords.length + jobRecords.length;
+  const totalRecords = employeeRecords!.length + jobRecords!.length;
```

Comment:
Are we sure we have records here, or should we be handling this case specifically?

Reply:
made code more explicit